tests/syntax/test_specifiers.py

```diff
@@ -505,12 +505,44 @@ def test_visible():
         "             with visibleDistance 10, with viewAngle 90 deg\n"
         "ego = new Object visible"
     )
+    radius = math.sqrt(3 * 0.5**2)
+    for i in range(30):
+        scene = sampleScene(scenario, maxIterations=100)
+        ego, base = scene.objects
+        assert ego.position.distanceTo(base.position) <= 10 + radius
+        assert ego.position.x >= base.position.x - radius
+        assert ego.position.y >= base.position.y - radius
+
+
+def test_visible_2d():
+    scenario = compileScenic(
+        "ego = new Object at 100 @ 200, facing -45 deg,\n"
+        "             with visibleDistance 10, with viewAngle 90 deg\n"
+        "ego = new Object visible",
```

Comment:
New/modified tests should use multiline strings (this old code dates to before `compileScenic` could properly remove the indentation resulting from their use).